src/molecules/gv-option/gv-option.js

```diff
@@ -125,6 +126,21 @@ export class GvOption extends LitElement {
         gv-button.description {
           margin: 0.5rem;
         }
+
+        .enterprise-lock__container {
+          position: absolute;
+          top: 1px;
+          right: 1px;
+          padding: 6px 6px 16px 16px;
+          border-radius: 0 0 0 38px;
+          background: #e7e8ef;
+          z-index: 2;
```

Comment:
had to do this because of the cache icon (it's too big) 🙈

Reply:
🤷‍♂️